window/src/lib.rs

```diff
@@ -64,6 +64,12 @@ pub enum MouseCursor {
     SizeLeftRight,
 }
 
+pub enum TabBarItem {
```

Comment:
I don't feel great about introducing a tab bar item type here, because the `window` crate doesn't really do anything with tabs.

See my other comment: I think we can eliminate this from this PR!